lib/v2/douban/other/explore.js

```diff
@@ -10,22 +10,37 @@ module.exports = async (ctx) => {
     const data = response.data;
 
     const $ = cheerio.load(data);
-    const list = $('div[data-item_id]');
-    let itemPicUrl;
+    const list = $('div.item');
 
     ctx.state.data = {
         title: '豆瓣-浏览发现',
         link: 'https://www.douban.com/explore',
         item:
             list &&
             list
-                .map((index, item) => {
+                .map((_, item) => {
                     item = $(item);
-                    itemPicUrl = item.find('a.cover').attr('style').replace('background-image:url(', '').replace(')', '');
+
+                    const haveItemPic = item.find('a.cover').attr('style');
+                    const itemPicHtml = haveItemPic
+                        ? `<img src=${
+                              item
+                                  .find('a.cover')
+                                  .attr('style')
+                                  .match(/\('(.*?)'\)/)[1]
+                          }>`
+                        : '';
+
+                    const title = item.find('.title a').first().text() ? item.find('.title a').first().text() : '#' + item.find('.icon-topic').text();
+                    const desc = item.find('.content p').text();
+                    const author = item.find('.usr-pic a').last().text();
+                    const link = item.find('.title a').attr('href') ?? item.find('.icon-topic a').attr('href');
+
                     return {
-                        title: item.find('.title a').first().text(),
-                        description: `作者：${item.find('.usr-pic a').last().text()}<br>描述：${item.find('.content p').text()}<br><img src="${itemPicUrl}">`,
-                        link: item.find('.title a').attr('href'),
+                        title,
+                        author,
+                        description: `作者：${author}<br>描述：${desc}<br>${itemPicHtml}`,
```

Comment:
Please use [art-template](https://docs.rsshub.app/joinus/script-standard.html#template) for rendering HTML https://github.com/DIYgod/RSSHub/discussions/8359